#获取目标位置的服务
# empty request target pose
string world_frame_id
string ar_tag_frame_id
shape_msgs/SolidPrimitive shape
geometry_msgs/Pose[] remove_at_poses
---
# target pose response
bool succeeded
geometry_msgs/Pose target_pose

